### PR TITLE
Update audio-capture.md

### PR DESCRIPTION
**Pull request by kristofmuhi**
added more explicit callout on what is supported and what is not

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [ ] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/main/docs/contributing.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/mobile-apps/features/audio-capture.md [hunk 1]
```diff
@@ -4,7 +4,9 @@ title: Audio Capture
 ---
 import useBaseUrl from '@docusaurus/useBaseUrl';
 
-<p><small><span className="sauceDBlue">Real Devices Only</span></small></p>
+<p><small><span className="sauceDBlue">
```
Comment (lysannep) [inline docs/mobile-apps/features/audio-capture.md:7]: The badge now says "Devices Only" -- Is that intended?

File: docs/mobile-apps/features/audio-capture.md [hunk 2]
```diff
@@ -73,7 +75,9 @@ curl --compressed \ -O https://{SAUCE_USERNAME}:{SAUCE_ACCESS_KEY}@{DATA_CENTER}
 ## Limitations
 
 :::note Not Supported
-* Emulators and Simulators are not yet supported. 
+* Real time audio capture is not yet supported. 
+* Accessibility and TalkBack on Android are not supported.
+* Emulators and Simulators are not supported.
```
Comment (lysannep) [inline docs/mobile-apps/features/audio-capture.md:80]:
```suggestion
* Emulators and simulators are not supported. 
```

**Review by lysannep — CHANGES_REQUESTED** [review]
Added a few comments. Thank you!

**Comment by kristofmuhi** [comment]
> @aoleszyn ~ The `audio-capture.md` file hasn't been added to `sidebar.js,` so it doesn't show up in the side-navigation. We should fix that.

File: docs/mobile-apps/features/audio-capture.md (hunk 1 above)
Comment (kristofmuhi) [reply docs/mobile-apps/features/audio-capture.md:7]: it is not!

**Comment by kristofmuhi** [comment]
@lysannep can you please help to make this page visible on the page and not hidden since @aoleszyn is out this week?

**Comment by lysannep** [comment]
> 

@kristofmuhi ~ Done! I made the change in the audio-capture-callout branch.

**Review by spider-sauce — APPROVED** [review]

**Comment by aoleszyn** [comment]
> @aoleszyn ~ The `audio-capture.md` file hasn't been added to `sidebar.js,` so it doesn't show up in the side-navigation. We should fix that.

@lysannep  Originally it was meant to be "hidden". The publishing of the page is scheduled.

**Review by aoleszyn — APPROVED** [review]

**Pull request merged**